homework9/src/components/Subscribe/Subscribe.js

```diff
@@ -1,9 +1,13 @@
 import React from "react";
 
-import Button from "./Button";
-import DescriptionText from "./DescriptionText";
+import DescriptionText from "../DescriptionText/DescriptionText";
+import SubscribeBtn from "../SubscribeBtn/SubscribeBtn";
+import SubscribeModal from "../SubscribeModal/SubscribeModal";
+
+import "./Subscribe.css";
 
 function Subscribe() {
+  const [open, setOpen] = React.useState(false);
```

Comment:
u can use `useState` instead of `React.useState`

import React, { useState } from 'react'